_posts/2023-12-28-release.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Release 2023.52
+author: Priya
+date: 2023-12-28
+category: Jekyll
+layout: post
+---
+### Enhancements:
+#### BigQuery export - partitioning on date columns.
+Users can now enable partitioning on date columns when exporting to BigQuery for enhanced efficiency and cost savings.
+
+### Defect Fixes:
+* Resolved an issue where a continuous loader was displayed on the Data landing page when a user, logged into multiple workspaces and attempted to log in after logging out from another workspace.
+* Resolved issues related to the propagation and renaming of column names within the joins workflow.
```

Comment:
Resolved issues related to the propagation and renaming of column names in Join rule.